TrafficCapture/trafficCaptureProxyServerTest/src/main/docker/docker-compose.yml

```diff
@@ -1,6 +1,30 @@
 version: '3.7'
 services:
   webserver:
-    image: 'nginx-simple:latest'
+    image: 'migrations/nginx:latest'
```

Comment:
We should probably choose a more descriptive name than nginx.  For this project, it is fine, but the image will live in a more general migrations space.  One might presume that it's a more general purpose nginx container, but it's instead highly tailored to this testing framework.  Something like nginx-perf-test-webserver.

Reply:
Oh I thought you meant the name of the container, not the image. I'll update that as you recommended.